tests/test_fits.py

```diff
@@ -128,6 +128,22 @@ def testUndefinedVector(self):
         metadata = self.writeAndRead(header)
         self.assertEqual(metadata.getArray("FOO"), [None, None])
 
+    def testNamedHeaderNavigate(self):
```

Comment:
Am I missing a test for the python setHdu interface?

Reply:
No. {{getHdu}} and other functions exported to python don't have a test either.

Reply:
no, you are not missing it I didn't write one... but I can if needed. 

I didn't find one anywhere else either.

Reply:
so this is resolved (I think) by https://github.com/lsst/afw/pull/657 being done